docker/jenkins/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+[ -d build ] && rm -rf build
+mkdir build && cd build
+cmake \
+  -D CMAKE_CXX_FLAGS="-Wall -Wextra" \
+  -D CMAKE_CXX_COMPILER=g++ \
+  -D PYTHON_EXECUTABLE=/usr/bin/python3 \
+  -D Tasmanian_ENABLE_MATLAB=OFF \
+  ..
+make -j${NPROC}
+ctest -j${NPROC} --no-compress-output -T Test
```

Comment:
Do not use "ctest -j***". By default, Tasmanian uses OpenMP and -j can lead to more threads than cores and end up with slower runtime. Also, there are only 4 ctests (if you enable Fortran), multi-threading will probably not help here.

Reply:
With CMake you can set properties on a test and specify how many process slots are required.